common/common.scss

```diff
@@ -4,360 +4,44 @@
-  border-right: none;
-}
-@media (min-width: 1200px) {
-  .header__navItem {
-    margin-right: 20px;
-    padding-right: 20px;
-  }
-}
-
-.header__link {
-  color: #383838;
-  display: block;
-  float: inherit;
-}
-
-.header__link:hover, .header__link:focus {
-  color: #d57800 !important;
-  text-decoration: none;
-}
-
-.header__link:visited {
-  color: #383838;
-}
-
-@media (min-width: 960px) {
-  .header__link {
-    display: inline;
-  }
-}
-
-.header__avatar {
-  width: 22px;
-  height: 22px;
-  border-radius: 50%;
-  margin-bottom: 13px;
-}
-
-.header__cart:hover, .header__cart:focus {
-  text-decoration: none;
-}
-
-.header__itemBadge {
-  display: inline-block;
-  background: #FFA400;
-  padding: 0 2px;
-  min-width: 20px;
-  height: 20px;
-  font-size: 14px;
-  border-radius: 20px;
-  line-height: 22px;
-  position: relative;
-  text-align: center;
-  top: -16px;
-  left: -10px;
-  color: #fff;
-  margin-right: -15px;
-}
 
-.nav-icon {
-  width: 22px;
-  height: 22px;
-}
-.nav-icon path, .nav-icon circle {
-  fill: #666666;
-}
-.nav-icon:hover path, .nav-icon:hover circle {
-  fill: #7D55C7;
-}
-
-#header.header--inverted {
-  border-bottom: 0;
-  position: absolute;
-  top: 0px;
-  left: 0px;
-}
-@media (min-width: 960px) {
-  .mobileHeader {
-    display: none;
-  }
-
-  .header {
-    display: block;
-  }
-}
-@media print {
-  .mobileHeader {
-    display: none;
-  }
-}
-
-.mobileHeader__container {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding: 10px 15px;
-  border-bottom: 1px solid #CECECE;
+html, body {
+  font-family: "Lato", helvetica, sans-serif;
 }
 
-.mobileHeader__icon {
-  font-size: 24px;
-  width: 20px;
-  color: #383838;
+nav.navbar {
+  background-color: #034278;
+  color: white;
+  padding: 1rem 0;
 }
 
-.mobileHeader__logo {
-  max-width: 180px;
-  width: 100%;
-
+.navbar-brand {
   img {
-    width: auto;
-    height: auto;
-    max-width: 100%;
-    vertical-align: middle;
-    border: 0;
-  }
-}
-@media (min-width: 500px) {
-  .mobileHeader__logo {
-    max-width: 250px;
+    height: 1em;
   }
 }
 
-.mobileHeader__dropdown {
-  display: none;
-  background: #eee;
-  padding: 30px 15px;
-}
-
-.mobileHeader__links {
-  background: #fff;
-  margin: 0;
-  list-style-type: none;
-  box-shadow: 0px 2px 6px #CECECE;
-  text-transform: uppercase;
-}
+ul.navbar-nav {
+  li.nav-item  {
+    > a {
+      color: white;
 
-.mobileHeader__shop {
-  display: flex;
-  justify-content: space-between;
-}
-.mobileHeader__shop:hover, .mobileHeader__shop:focus {
-  color: #383838;
-}
-
-.mobileHeader__links--downcase {
-  display: none;
-  text-transform: none;
-  list-style-type: none;
-  margin: 0;
+      &:not(.btn):hover {
+        color: white;
+        opacity: 0.8;
+        transform: translate(0, 1px);
+      }
+    }
+  }
 }
 
-.mobileHeader__dropdownItem {
-  padding: 10px 0;
+// Bootstrap .row class overrides the built in discourse .row class, so we need to override it again
```

Comment:
This is now necessary as we moved from bootstrap 3 to bootstrap 5. There might be an easier way to do the conflicts but we do want to get the header out the door quick